### PR TITLE
Add flare series section, enhancements

### PR DESCRIPTION
**Pull request by SYangster**
### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: web/src/components/tutorials.astro [hunk 1]
```diff
@@ -182,8 +187,8 @@ const tutorials = [
     link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/ml-to-fl"
   },
   {
-    title: "Hello FedAvg Numpy",
-    tags: ["beg.", "algorithm", "numpy", "model-controller"],
+    title: "Hello FedAvg",
+    tags: ["beg.", "algorithm", "pytorch", "model-controller"],
     description: "Example using the FedAvg workflow to implement Federated Averaging.",
     link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/hello-fedavg/README.md"
```
Comment (YuanTingHsieh) [inline web/src/components/tutorials.astro:193]: no related to this PR, maybe there is somewhere we could use variable substitution or something for these stuff?
<github link [2.5]> or something.

File: web/src/components/series.astro [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+          tags: ["docs"],
+          description: "The FLARE ModelController API provides an easy way for users to write and customize FLModel-based controller workflows.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/controllers/model_controller.html#model-controller"
+        },
+        {
+          title: "Fed Job API",
+          tags: ["docs"],
+          description: "The FLARE FedJob API allows users to Pythonically define and create job configurations.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/fed_job_api.html",
+          colab_link: "https://colab.research.google.com/github/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_api.ipynb"
```
Comment (YuanTingHsieh) [inline web/src/components/series.astro:34]: this has a colab link though, is it "docs"?

File: web/src/components/series.astro [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+        },
+        {
+          title: "Hello FedAvg",
+          tags: ["beg.", "pytorch"],
+          description: "Demonstrate flexibility of the ModelController API, and show how to write a Federated Averaging workflow with early stopping, model selection, and saving and loading.",
+          link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/hello-fedavg/README.md"
+        },
+        {
+          title: "Job API Examples",
+          tags: ["int.", "pytorch", "lightning", "sklearn", "tensorflow"],
```
Comment (YuanTingHsieh) [inline web/src/components/series.astro:50]: int. means "intermediate"? or "integration"?

File: web/src/components/series.astro [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+  title: "FLARE 200: Algorithms",
+  subseries_list: [
+    {
+      id: "201",
+      title: "201: FL Algorithms",
+      description: "See how FL algorithms can be implemented in a variety of frameworks.",
+      cards: [
+        {
+          title: "Step-by-Step CIFAR10 Examples",
+          tags: ["beg.", "algorithm", "pytorch", "dl"],
```
Comment (YuanTingHsieh) [inline web/src/components/series.astro:153]: what things are categorized as tag "algorithm"? we could also say all the examples are using different types of algorithm

File: web/src/components/tutorials.astro (hunk 1 above)
Comment (SYangster) [reply web/src/components/tutorials.astro:193]: yes good idea, I can later look to add substitution for the links and automatic versioning for the site content

File: web/src/layouts/Layout.astro [hunk 5]
```diff
@@ -19,7 +19,7 @@ const makeTitle = title ? title + " | " + "NVIDIA FLARE" : "NVIDIA FLARE";
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
-    <link rel="icon" type="image/x-icon" href="/favicon.ico">
+    <link rel="icon" type="image/x-icon" href="public/favicon.ico">
```
Comment (YuanTingHsieh) [inline web/src/layouts/Layout.astro:22]: do we need public

File: web/src/components/series.astro (hunk 3 above)
Comment (SYangster) [reply web/src/components/series.astro:50]: intermediate. These tags are from the tutorial catalog page.

File: web/src/components/series.astro (hunk 4 above)
Comment (SYangster) [reply web/src/components/series.astro:153]: "algorithm" was from the tags in the tutorial catalog page, where we divided all examples in "algorithm", "tool", 
"deployment"

File: web/src/components/series.astro (hunk 2 above)
Comment (SYangster) [reply web/src/components/series.astro:34]: good catch, will remove

File: web/src/components/series.astro (hunk 4 above)
Comment (YuanTingHsieh) [reply web/src/components/series.astro:152]: got it, the series.astro now just want to be consistent with tutorial catalog page.
makes sense, we can thinking of any tagging enhancements in other PRs.

File: web/src/components/tutorials.astro [hunk 6]
```diff
@@ -91,6 +89,13 @@ const tutorials = [
     description: "Example integrating NVIDIA FLARE with Weights and Biases streaming capability from clients to the server.",
     link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/experiment-tracking/wandb/README.md"
   },
+  {
+    title: "Containerized Deployment with Docker",
+    tags: ["beg.", "deployment"],
```
Comment (YuanTingHsieh) [inline web/src/components/tutorials.astro:94]: in another PR, maybe we need "doc" for this one not sure

**Review by YuanTingHsieh — APPROVED** [review]

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**